docs/advanced-usage.md

```diff
@@ -31,6 +31,36 @@ fail or not.
 If you don't specify a `penalty` configuration, the plugin will use the [default threshold values][penaltyextensioncode], which are to
 allow any warning, but break the build on any error.
 
+#### Default `penalty` profiles
```

Comment:
Two notes here:
1. Why is this an h4 and not h3? It is a sub-section of an h2, should be an h3 then
2. Can we put the various built-ins as a bulleted list? It's a bit unclear which text refers to which snippet right now

Reply:
First is done. I'm not sure how I will put `bullet`s here. Can you maybe modify to help this? You can even do that through the Github UI in this PR. Thanks.

Reply:
Sure! On it ;)

Reply:
05c9492 done!

Reply:
Thanks 👍 Much clear now. Especially the `(Default)` indicator.